modules/_labs/core/react/lib/CanvasProvider.tsx

```diff
@@ -16,8 +17,10 @@ export default class CanvasProvider extends React.Component<CanvasProviderProps>
     const {children, theme} = this.props;
 
     return (
-      <ThemeProvider value={theme}>
-        <InputProvider>{children}</InputProvider>
+      <ThemeProvider theme={theme}>
+        <div dir={theme.direction}>
```

Comment:
Can we use a [`bdo` element](https://css-tricks.com/almanac/properties/d/direction/) instead of a `div`? Also, I think we should include css `direction` as a backup since it has better support.

```tsx
const DirectionContainer = styled('bdo')(({theme}) => ({direction: theme.direction}))
```

Reply:
👍